blockchain_api/app.js

```diff
@@ -40,6 +41,10 @@ var invoke = require('./app/invoke-transaction.js');
 var query = require('./app/query.js');
 var host = process.env.HOST || hfc.getConfigSetting('host');
 var port = process.env.PORT || hfc.getConfigSetting('port');
+
+var peers = ["peer0.org1.example.com","peer1.org1.example.com"];
+var chaincodeName = 'mycc11';
+var channelName = 'mychannel';
```

Comment:
These parameters should also be configurable, we will have peers like `peer0.fabric.opetbot.com` and `opet_channel` channel.

Reply:
For the chaincode, let's also use something meaningfull, like `opet_cc`.